charts/fint-adapter-sikri/values-innlandet.yaml

```diff
@@ -0,0 +1,157 @@
+# Default values for fint-adapter-sikri.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+nameOverride: adapter-sikri-arkiv-innlandetfylke
+
+onePasswordVault: 
```

Comment:
This is not a vault but an item. It should probably be called `onePasswordItem`.